### PR TITLE
Add check_server_rack_temperature script

### PR DESCRIPTION
**Pull request by Scriptkiddi**

### CODE REVIEW

File: files/check_server_rack_humidity [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Checks the temperature in the server rack and outputs the results as perfdata.
```
Comment (dasJ) [inline files/check_server_rack_humidity]: Using what? Please specify the hardware

File: files/check_server_rack_humidity [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+			usage
+			;;
+		:)
+			echo "Option -${OPTARG} requires an argument"
+			usage
+			;;
+	esac
+done
+
+humidity=$(bc <<< "$($(dirname "${0}")/tinkerforge --host $host call humidity-v2-bricklet $uid get-humidity | cut -d "=" -f 2)/100")
```
Comment (dasJ) [inline files/check_server_rack_humidity]: Please quote

File: files/check_server_rack_humidity [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+# Format output
+if [ "${humidity}" -gt "${crit}" ]; then
+	checkState=CRITICAL
+elif [ "${humidity}" -gt "${warn}" ]; then
+	checkState=WARNING
+fi
+checkOutput="${humidity}%"
```
Comment (dasJ) [inline files/check_server_rack_humidity]: Please add some text instead of a single number

File: files/check_server_rack_humidity (hunk 1 above)
Comment (dasJ) [reply files/check_server_rack_humidity]: Also: temperature != humidity

File: files/check_server_rack_intake_temperature [hunk 4]
```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Checks the temperature in the server rack and outputs the results as perfdata.
```
Comment (dasJ) [inline files/check_server_rack_intake_temperature]: Using what hardware?

File: files/check_server_rack_humidity [hunk 5]
```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Checks the temperature in the server rack and outputs the results as perfdata.
+#
+# Parameters:
+# -c - Temperature that is critical (defaults to 40°C)
```
Comment (dasJ) [inline files/check_server_rack_humidity]: Temperature != humidity

File: files/check_server_rack_intake_temperature [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+			usage
+			;;
+		:)
+			echo "Option -${OPTARG} requires an argument"
+			usage
+			;;
+	esac
+done
+
+temp=$(bc <<< "$($(dirname "${0}")/tinkerforge --host $host call  ptc-v2-bricklet $uid get-temperature | cut -d "=" -f 2)/100")
```
Comment (dasJ) [inline files/check_server_rack_intake_temperature]: Please quote

File: files/check_server_rack_intake_temperature [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+	checkState=CRITICAL
+elif [ "${temp}" -gt "${warn}" ]; then
+	checkState=WARNING
+fi
+checkOutput="${temp} °C"
+
+# Format perfdata
+checkPerfdata="temperature=${temp}°C;${warn};${crit}"
+
+checkReturn
```
Comment (dasJ) [inline files/check_server_rack_intake_temperature]: Thinking about it, please merge the two scripts. Something like `check_tinkerforge` where you can give a type `-t` and hence select what to monitor

File: files/check_server_rack_temperature [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+	checkState=CRITICAL
+elif [ "${temp}" -gt "${warn}" ]; then
+	checkState=WARNING
+fi
+checkOutput="${temp} °C"
+
+# Format perfdata
+checkPerfdata="temperature=${temp}°C;${warn};${crit}"
+
+checkReturn
```
Comment (dasJ) [inline files/check_server_rack_temperature:66]: I stand corrected. It's three scripts that should be merged. Especially the two temperature scripts. Copypasting everything is not a good solution

File: tasks/main.yml [hunk 9]
```diff
@@ -29,6 +29,16 @@
...
 
+- name: Copy check scripts dependencies
+  copy:
+    src: "{{ item }}"
+    dest: "{{ icinga2_scripts_path }}"
+    mode: 0755
+    owner: root
+    group: root
+  with_items:
+    - tinkerforge
```
Comment (dasJ) [inline tasks/main.yml:40]: Maybe `when: needs_tinkerforge`. This way, we don't push the file to ALL servers

File: tasks/main.yml [hunk 10]
```diff
@@ -53,6 +63,9 @@
     - zfs
     - zfs_snapshot
     - zpool_scrub
+    - server_rack_temperature
+    - server_rack_intake_temperature
+    - server_rack_humidity
```
Comment (dasJ) [inline tasks/main.yml]: That's not even close to alphabetical

**Review by dasJ — CHANGES_REQUESTED** [review]

File: files/check_server_rack_humidity (hunk 3 above)
Comment (Scriptkiddi) [reply files/check_server_rack_humidity]: Humidity is measured in percent what would you suggest?

File: files/check_server_rack_temperature (hunk 8 above)
Comment (Scriptkiddi) [reply files/check_server_rack_temperature:66]: i would like to have seperate scripts for different measurment units

File: files/check_server_rack_humidity (hunk 3 above)
Comment (SlothOfAnarchy) [reply files/check_server_rack_humidity]: checkOutput is a short summary of the check, most times a full sentence.
```suggestion
checkOutput="Humidity is ${humidity}%"
```

**Comment by Scriptkiddi** [comment]
@dasJ willst hier mal drüber schauen?

**Pull request closed without merging**